<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QWidget" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ํ์๊ฐ์</string>
  </property>
  <widget class="QLineEdit" name="name_signup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>์ด๋ฆ์ ์๋ฅํ์ธ์</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupPwShow_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>๐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ํ์๊ฐ์</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="accnum_signup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>๊ณ์ข๋ฒํธ๋ฅผ ์๋ฅํ์ธ์</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_signup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>์์ด๋๋ฅผ ์๋ฅํ์ธ์</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_signup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>๋น๋ฐ๋ฒํธ๋ฅผ ์๋ฅํ์ธ์</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ํ์๊ฐ์</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
